src/main/java/module-info.java

```diff
@@ -61,7 +61,11 @@
     requires jbibtex;
     requires citeproc.java;
     requires antlr.runtime;
-    requires org.graalvm.js;
+    requires org.graalvm.truffle;
+    requires org.graalvm.sdk;
+    requires transitive org.graalvm.js;
+    requires java.scripting;
```

Comment:
I thought that `2.1.0-SNAPSHOT` does not require JavaScript anymore (see also https://github.com/michel-kraemer/citeproc-java/issues/92). @tobiasdiez Do you remember why you had to add these dependencies?